application/src/main/java/org/opentripplanner/updater/AllowedRentalType.java

```diff
@@ -0,0 +1,7 @@
+package org.opentripplanner.updater;
+
+public enum AllowedRentalType {
```

Comment:
This class should be inside org.opentripplanner.updater.vehicle_rental.datasources.params package.